### ISSUE TITLE
File conflict between aliyun-python-sdk-dysmsapi and aliyun-python-sdk-dybaseapi

### ISSUE THREAD

**Issue by glaubitz** [aliyun/aliyun-openapi-python-sdk#357]
I'm currently packaging the Python Alibaba SDK for openSUSE/SLE and I have run into some issues, in this case with the packages ```aliyun-python-sdk-dysmsapi``` and ```aliyun-python-sdk-dybaseapi```.

Both packages are trying to install the file ```/usr/lib/python3.8/site-packages/aliyunsdkdybaseapi/__init__.py``` (see RPM output below):

```
File /usr/lib/$PYTHONVER/site-packages/aliyunsdkdybaseapi/__init__.py
  from install of
     python3-aliyun-python-sdk-dysmsapi-1.0.0-2.1.noarch (Branch project for package python-aliyun-python-sdk-aas (openSUSE_Tumbleweed))
  conflicts with file from install of
     python3-aliyun-python-sdk-dybaseapi-1.0.4-2.1.noarch (Branch project for package python-aliyun-python-sdk-aas (openSUSE_Tumbleweed))
```

This prevents the co-installation of both packages.

If both packages are supposed to use same directory, I recommend using the namespace feature (implicit namespaces for Python 3), similar to what Microsoft is using for it's Python Azure SDK (see: https://github.com/Azure/azure-sdk-for-python).

**Comment by glaubitz** [comment]
It seems that only the package on PyPi is affected by this problem. The git repository does not have the ```aliyunsdkdybaseapi``` folder, see:

> https://github.com/aliyun/aliyun-openapi-python-sdk/tree/master/aliyun-python-sdk-dysmsapi

Could you release a new version of the package on PyPi to fix this problem?

**Comment by glaubitz** [comment]
I suggest releasing new versions of all the module packages in the tree. This would also fix #356.

**Comment by 824750130** [comment]
New version has been released,If you have any questions, please reopen it.
